config/initializers/canonical_rails.rb

```diff
@@ -9,8 +9,7 @@
 
   # This is the main host, not just the TLD, omit slashes and protocol. If you have more than one, pick the one you want to rank in search results.
 
-  # TODO: update when we have domain
-  config.host = "www.education.gov.uk"
+  config.host = "manage-training-for-early-career-teachers.education.gov.uk"
```

Comment:
For my understanding, is this (or should it be) environment-dependant?

Reply:
I really don't know to be honest. I think having it wrong in prod is a _bad thing_, but having it wrong elsewhere isn't an issue. We don't want staging to appear in search results